warehouse/templates/manage/account/publishing.html

```diff
@@ -34,7 +34,8 @@ <h3>Projects with active publishers</h3>
   {% set projects = [] %}
   {% for project in user.projects %}
     {% if project.oidc_publishers %}
-      {{ projects.append(project) }}
+      {# NOTE: `or ""` to prevent rendering None #}
+      {{ projects.append(project) or "" }}
     {% endif %}
   {% endfor %}
```

Comment:
I think the Jinja-ish way to do this would be something like:

{% set projects = user.projects | selectattr('oidc_publishers') %}

Reply:
Whoops, you're completely right (and that's way nicer). I'll update now.